chimeraos/airootfs/root/install.sh

```diff
@@ -34,6 +34,46 @@ if [ -d "/etc/NetworkManager/system-connections" ]; then
         ${MOUNT_PATH}/etc/NetworkManager/system-connections/.
 fi
 
+# Detect hybrid intel-nvidia setups
+NVIDIA_BUSID=$(lspci -nm -d 10de: | \
+    awk '{print $1 " " $2 " " $3}' | \
+    grep -e 300 -e 302 | \
+    awk '{print $1}' | \
+    sed 's/\./:/' )
+INTEL_BUSID=$(lspci -nm -d 8086: | \
+    awk '{print $1 " " $2 " " $3}' | \
+    grep -e 300 -e 302 | \
+    awk '{print $1}' | \
+    sed 's/\./:/' )
+
+if [[ $INTEL_BUSID == ??:??:? && $NVIDIA_BUSID == ??:??:? ]] ; then
+    if (whiptail --yesno "Intel/Nvidia hybrid setup is detected. Would you like to use this configuration?"); then
```

Comment:
Would this be clearer?

"Intel/Nvidia hybrid graphics detected. Would you like to force use of Nvidia graphics?"

Reply:
Sure